File: Problem/Easy/Problem_5/Solution.py
# Python3 program using to convert the given decimal number into an equivalent binary number. 

# Recursive function
def convertDecToBin(dec : int) -> int:
    if dec == 0:
        return 0
    return (dec % 2 + 10 * convertDecToBin(int(dec // 2)))

# Main
if __name__ == "__main__":
    dec = 10
    print(f'Convert {dec} to binary: {convertDecToBin(dec)}')

File: Problem/Easy/Problem_5/Solution.cpp
// C++ program using to convert decimal to binary number using recursion

#include <iostream>

// Recursive function
long long convertDecimalToBinary(int decimal) {
    if (decimal == 0)
        return 0;
    return (decimal % 2 + 10 * convertDecimalToBinary(decimal / 2));
}

// Main function
int main()
{
    int d = 10;
    std::cout << convertDecimalToBinary(d);
    return 0;
}